examples/cusparselt/matmul.py

```diff
@@ -73,15 +67,31 @@
 #
 # compresses the matrix A
 #
-compressed_size = cusparselt.spMMACompressedSize(handle, plan)
+compressed_size = numpy.array(1, dtype='uint32')
+compressed_buffer_size = numpy.array(1, dtype='uint32')
+cusparselt.spMMACompressedSize(handle, plan, compressed_size.ctypes.data,
+                               compressed_buffer_size.ctypes.data)
```

Comment:
I think this is wrong.

1. the `xxxx_size` values are the integer sizes, not the data arrays themselves
2. the `*CompressedSize*`  functions should be returning these sizes, not take anything as inputs.